
 
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 
 const credential = firebase.auth.PhoneAuthProvider.credential(verificationId, verificationCode);
@@ -364,7 +364,7 @@ the token and secret are returned for use with Twitter User API.
 #### Sign in
 
 ```typescript
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import {cfaSignIn, SignInResult} from 'capacitor-firebase-auth/alternative'; import {SignInResult} from './definitions';
 
 cfaSignIn('twitter.com').subscribe(
@@ -378,7 +378,7 @@ cfaSignIn('twitter.com').subscribe(
 #### Sign In using direct twitter method
 
 ```typescript
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import {cfaSignInTwitter, TwitterSignInResult} from 'capacitor-firebase-auth/alternative'; import {SignInResult} from './definitions';
 
 cfaSignInTwitter().subscribe(
diff --git a/node_modules/capacitor-firebase-auth/dist/esm/definitions.js b/node_modules/capacitor-firebase-auth/dist/esm/definitions.js
index e9055ab..e21d59c 100644
--- a/node_modules/capacitor-firebase-auth/dist/esm/definitions.js
+++ b/node_modules/capacitor-firebase-auth/dist/esm/definitions.js
@@ -1,4 +1,4 @@
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 export class GoogleSignInResult {
     constructor(idToken) {
diff --git a/node_modules/capacitor-firebase-auth/dist/esm/facades.js b/node_modules/capacitor-firebase-auth/dist/esm/facades.js
index 28bbfec..c9a3272 100644
--- a/node_modules/capacitor-firebase-auth/dist/esm/facades.js
+++ b/node_modules/capacitor-firebase-auth/dist/esm/facades.js
@@ -1,5 +1,5 @@
 import { registerWebPlugin, Plugins, Capacitor } from '@capacitor/core';
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 import { Observable, throwError } from 'rxjs';
 import { CapacitorFirebaseAuth } from './web';
diff --git a/node_modules/capacitor-firebase-auth/dist/esm/providers/facebook.provider.js b/node_modules/capacitor-firebase-auth/dist/esm/providers/facebook.provider.js
index 4f4863a..d97c48a 100644
--- a/node_modules/capacitor-firebase-auth/dist/esm/providers/facebook.provider.js
+++ b/node_modules/capacitor-firebase-auth/dist/esm/providers/facebook.provider.js
@@ -7,7 +7,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 import { FacebookSignInResult } from '../definitions';
 export const facebookSignInWeb = () => __awaiter(void 0, void 0, void 0, function* () {
diff --git a/node_modules/capacitor-firebase-auth/dist/esm/providers/google.provider.js b/node_modules/capacitor-firebase-auth/dist/esm/providers/google.provider.js
index 4ccbe57..1bc38e6 100644
--- a/node_modules/capacitor-firebase-auth/dist/esm/providers/google.provider.js
+++ b/node_modules/capacitor-firebase-auth/dist/esm/providers/google.provider.js
@@ -7,7 +7,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 import { GoogleSignInResult } from '../definitions';
 export const googleSignInWeb = () => __awaiter(void 0, void 0, void 0, function* () {
diff --git a/node_modules/capacitor-firebase-auth/dist/esm/providers/twitter.provider.js b/node_modules/capacitor-firebase-auth/dist/esm/providers/twitter.provider.js
index aa866c7..0b181ab 100644
--- a/node_modules/capacitor-firebase-auth/dist/esm/providers/twitter.provider.js
+++ b/node_modules/capacitor-firebase-auth/dist/esm/providers/twitter.provider.js
@@ -7,7 +7,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 import { TwitterSignInResult } from '../definitions';
 export const twitterSignInWeb = () => __awaiter(void 0, void 0, void 0, function* () {
diff --git a/node_modules/capacitor-firebase-auth/dist/esm/web.js b/node_modules/capacitor-firebase-auth/dist/esm/web.js
index c37ee1d..42fa9ad 100644
--- a/node_modules/capacitor-firebase-auth/dist/esm/web.js
+++ b/node_modules/capacitor-firebase-auth/dist/esm/web.js
@@ -8,7 +8,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
     });
 };
 import { registerWebPlugin, WebPlugin } from '@capacitor/core';
-import * as firebase from 'firebase/app';
+import firebase from 'firebase/app';
 import 'firebase/auth';
 import { facebookSignInWeb } from './providers/facebook.provider';
 import { googleSignInWeb } from './providers/google.provider';